code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -222,7 +222,7 @@
 	origin_tech = "combat=6;materials=1;syndicate=2"
 	slot_flags = SLOT_BACK
 	ammo_type = "/obj/item/ammo_casing/a762"
-	allowed_magazines = list(/obj/item/ammo_casing/a762)
+	allowed_magazines = list(/obj/item/ammo_magazine/a762)

```

Comment:
This was an accident, it is a fix because I put the ammo type instead of the magazine, so, that is breaking this gun in particular, also, not even worthy adding in the changelog since it is another admin bus thing.